### ISSUE TITLE
flake8 broken by pyrestcli's future==0.15.2 dependency

### ISSUE THREAD

**Issue by caktus-philip** [danicarrion/pyrestcli#20]
The version of `future` that `pyrestcli` currently uses (0.15.2) causes flake8 errors for us, although the connection is a little complicated. 

We added `carto-python` (https://github.com/CartoDB/carto-python/) to our project which depends on `pyrestcli`, and `pyrestcli` depends on `future==0.15.2`. This was the first package in our project to include `future`. 

When our continuous integration server tested the change, we suddenly got a lot of flake8 errors in files that should be excluded from flake8 coverage. We found out that this is a known problem when flake8 is run in an environment with `future` installed —
https://gitlab.com/pycqa/flake8/issues/221

The problem is related to future’s implementation of `configparser` which was removed in 0.16.0 —
https://github.com/PythonCharmers/python-future/issues/118

TL;DR - `pyrestcli` uses a version of `future` that breaks flake8; changing `pyrestcli` to use `future` >= 0.16.0 should solve the problem. 

I understand that you might have other reasons that you don't want to upgrade the `future` version.  Thanks in advance for your consideration.

**Comment by danicarrion** [comment]
Sorry I never saw this issue until now. I've relaxed the requirement on the version of `future` on 0.6.7. Thanks!
